src/WebApiToTypeScript/WebApiToTypeScript.cs

```diff
@@ -215,7 +217,7 @@ private Config.Config GetConfig(string configFilePath)
             foreach (var resourceConfig in config.ResourceConfigs)
                 resourceConfig.SourcePath = ToAbsolutePath(baseDir, resourceConfig.SourcePath);
 
-            return config;
+            return config.ApplyCommandLineConfiguration(CommandLineConfig);
```

Comment:
I think this applying command line configuration should happen right after ```var baseDir = Path.GetFullPath(Path.GetDirectoryName(configFilePath) ?? "");``` -- so that the rest of the absolute path calculations, etc can continue to happen, and also for other overrides we might have?

Reply:
Good suggestion! 

Please check my updated PR, I did some changes to support both relative and absolute paths now as well